plugins/pci-psp/fu-plugin-pci-psp.c

```diff
@@ -8,279 +8,18 @@
-	fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS);
-}
-
-static void
-fu_plugin_add_security_attrs_rom_armor(FuPlugin *plugin, const gchar *path, FuSecurityAttrs *attrs)
-{
-	g_autoptr(FwupdSecurityAttr) attr = NULL;
-	g_autoptr(GError) error_local = NULL;
-	gboolean val;
-
-	/* create attr */
-	attr = fu_plugin_security_attr_new(plugin, FWUPD_SECURITY_ATTR_ID_AMD_SPI_WRITE_PROTECTION);
-	fu_security_attrs_append(attrs, attr);
-
-	if (!fu_plugin_pci_psp_get_attr(attr, path, "rom_armor_enforced", &val, &error_local)) {
-		g_debug("%s", error_local->message);
-		return;
-	}
-
-	if (!val) {
-		g_debug("ROM armor not enforced");
-		fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_NOT_ENABLED);
-		fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_ACTION_CONTACT_OEM);
-		return;
-	}
-
-	/* success */
-	fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_ENABLED);
-	fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS);
-}
-
-static void
-fu_plugin_add_security_attrs_rpmc(FuPlugin *plugin, const gchar *path, FuSecurityAttrs *attrs)
-{
-	g_autoptr(FwupdSecurityAttr) attr = NULL;
-	g_autoptr(GError) error_local = NULL;
-	gboolean val;
-
-	/* create attr */
-	attr =
-	    fu_plugin_security_attr_new(plugin, FWUPD_SECURITY_ATTR_ID_AMD_SPI_REPLAY_PROTECTION);
-	fu_security_attrs_append(attrs, attr);
-
-	if (!fu_plugin_pci_psp_get_attr(attr, path, "rpmc_spirom_available", &val, &error_local)) {
-		g_debug("%s", error_local->message);
-		return;
-	}
-
-	if (!val) {
-		g_debug("no RPMC compatible SPI rom present");
-		fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_NOT_SUPPORTED);
-		return;
-	}
-
-	if (!fu_plugin_pci_psp_get_attr(attr,
-					path,
-					"rpmc_production_enabled",
-					&val,
-					&error_local)) {
-		g_debug("%s", error_local->message);
-		return;
-	}
-
-	if (!val) {
-		g_debug("no RPMC compatible SPI rom present");
-		fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_NOT_ENABLED);
-		return;
-	}
-
-	/* success */
-	fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_ENABLED);
-	fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS);
-}
-
-static void
-fu_plugin_pci_psp_set_missing_data(FuPlugin *plugin, FuSecurityAttrs *attrs)
-{
-	g_autoptr(FwupdSecurityAttr) attr = NULL;
-
-	attr = fu_plugin_security_attr_new(plugin, FWUPD_SECURITY_ATTR_ID_SUPPORTED_CPU);
-	fwupd_security_attr_add_obsolete(attr, "cpu");
-	fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_MISSING_DATA);
-	fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_ACTION_CONTACT_OEM);
-	fu_security_attrs_append(attrs, attr);
-}
-
-static void
-fu_plugin_pci_psp_add_security_attrs(FuPlugin *plugin, FuSecurityAttrs *attrs)
-{
-	FuDevice *psp_device = fu_plugin_cache_lookup(plugin, "pci-psp-device");
-	const gchar *sysfs_path = NULL;
-	g_autofree gchar *test_file = NULL;
-
-	/* only AMD */
-	if (fu_cpu_get_vendor() != FU_CPU_VENDOR_AMD)
```

Comment:
we lost this, but I think that's okay

Reply:
Yeah I didn't think it was needed anymore since CCP only probes when right PCI device present.